website/source/docs/configuration/client.html.md

```diff
@@ -329,6 +333,18 @@ see the [drivers documentation](/docs/drivers/index.html).
   reserve on all fingerprinted network devices. Ranges can be specified by using
   a hyphen separated the two inclusive ends.
 
+
+### `template` Parameters
+
+- `function_blacklist` `([]string: ["plugin"])` - Specifies a list of template
+  rendering functions that should be disallowed in job specs because they can
+  leak information from the client host to templates.
```

Comment:
"leak information" is too weak to describe `plugin`, so maybe we should expand with:

> By default the `plugin` function is disallowed as it allows running arbitrary commands on the host as root (unless Nomad is configured to run as a non-root user).

Reply:
Done.